### PR TITLE
fix(argocd-image-updater): Bump application version

### PR DESCRIPTION
**Pull request by nielstenboom**
To support the changes made in https://github.com/argoproj/argo-helm/pull/987 , the application version has to be bumped to v0.10.3, as the current version does not have the aws cli embedded in the container yet.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argocd-image-updater/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The `config.registries` value can be used exactly as it looks in the documentati
 | fullnameOverride | string | `""` | Global fullname (argocd-image-updater.fullname in _helpers.tpl) override |
 | image.pullPolicy | string | `"Always"` | Default image pull policy |
 | image.repository | string | `"argoprojlabs/argocd-image-updater"` | Default image repository |
-| image.tag | string | `"v0.10.1"` | Default image tag |
+| image.tag | string | `"v0.10.3"` | Default image tag |
```
Comment (mkilchhofer) [inline charts/argocd-image-updater/README.md:76]:
```suggestion
| image.tag | string | `""` | Overrides the image tag whose default is the chart appVersion |
```

File: charts/argocd-image-updater/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: argocd-image-updater
 description: A Helm chart for Argo CD Image Updater, a tool to automatically update the container images of Kubernetes workloads which are managed by Argo CD
 type: application
-version: 0.4.0
+version: 0.4.1
 appVersion: v0.10.1
```
Comment (mkilchhofer) [inline charts/argocd-image-updater/Chart.yaml:6]:
```suggestion
appVersion: v0.10.3
```

File: charts/argocd-image-updater/values.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ image:
   # -- Default image pull policy
   pullPolicy: Always
   # -- Default image tag
```
Comment (mkilchhofer) [inline charts/argocd-image-updater/values.yaml:8]:
```suggestion
  # -- Overrides the image tag whose default is the chart appVersion
```

File: charts/argocd-image-updater/values.yaml [hunk 4]
```diff
@@ -6,7 +6,7 @@ image:
...
-  tag: v0.10.1
+  tag: v0.10.3
```
Comment (mkilchhofer) [inline charts/argocd-image-updater/values.yaml:9]:
```suggestion
  tag: ""
```

**Review by mkilchhofer — CHANGES_REQUESTED** [review]
Can we set the `image.tag` to empty string and only bump the charts appVersion?

**Review by mkilchhofer — APPROVED** [review]
lgtm. Thanks for your contribution 🙏

**Comment by mkilchhofer** [comment]
Can you please update against master branch? It doesn't matter if you rebase or merge master to your branch.

If you fork it to your personal account next time, it would help because then we are able to update your branch via GitHub UI. Unfortunately github does not support this for org accounts 😞

**Comment by nielstenboom** [comment]
Thanks for the review!

> If you fork it to your personal account next time, it would help because then we are able to update your branch via GitHub UI. Unfortunately github does not support this for org accounts 😞

Ah I wasn't aware, will do!

**Pull request merged**